index.js

```diff
@@ -151,11 +150,11 @@ module.exports = {
                     if (session) {
                         var filename = 'ScreenShot_onException-' + process.pid + '-' + new Date().getTime();
                         var screenShotFileName = path.resolve(screenShotPath, filename);
-                        flow.wait(function () {
-                            return nemo.screenshot.snap(screenShotFileName).then(function () {
-                                exception._nemoScreenshotHandled = true;
-                                throw exception;
-                            }, 10000);
+                        nemo.screenshot.snap(screenShotFileName).then(function () {
```

Comment:
I'm not sure the original code would have returned control to mocha (or wherever) even after 10 seconds. i didn't test that specifically. i'll outline my testing in the main PR comment.